test/serializer/instant-render/EmptyBuiltInProps.js

```diff
@@ -0,0 +1,19 @@
+// instant render
+// add at runtime:var __empty = {};
+
+function f(c) {
+  var a = {};
+  if (c) {
+    a.foo = 42;
+  }
+
+  return a;
+}
+
+global.__optimize && __optimize(f);
+
+inspect = function() {
+  let result = f(false);
+  if (!global.__optimize) result.foo = {};
```

Comment:
`inspect` will always run in node, where `global.__optimize` doesn't exist. So I think this makes this test useless.